docfx_yaml/extension.py

```diff
@@ -322,6 +322,10 @@ def build_finished(app, exception):
                 # Skip objects without a module
                 continue
 
+            if filename.lower().find('.tests') != -1:
+                # Skip *.tests*.yml files
+                continue
+
```

Comment:
after doing some research, we can exclude tests files by changing conf.py in CI:
http://www.sphinx-doc.org/en/stable/config.html#confval-exclude_patterns
so, rollback this change and will update CI.